py-rse-index.Rmd

```diff
@@ -72,100 +125,46 @@ Sami Virtanen
 
 -   A personal computer with Internet access.
--   A [Unix shell][unix-shell]
-    (see Appendix \@ref(py-rse-install) for how to set this up on Windows).
+-   A [Unix shell][unix-shell].
 -   [Python 3][python] and [Jupyter][jupyter].
--   [R][r] and [RStudio][rstudio].
 -   A [GitHub][github] account.
 
-If you are using Windows,
-please install [Chocolatey][chocolatey]:
-you can use it to install other programs as you go along.
-
-## What's the big picture? {#py-rse-intro-big-picture}
-
-To make sense of what follows,
-we must distinguish two key ideas:
-
--   [Open science][open-science]
-    focuses on making data, methods, and results freely available to all
-    by publishing them under [open licenses][open-license].
--   [Reproducible research][reproducible-research]
-    means ensuring that anyone with access to data and software can feasibly reproduce results,
-    both to check them and to build on them.
-
-People often conflate these terms,
-but they are distinct:
-
--   If you share your data and the programs that analyze it,
-    but don't document what steps to take in what order,
-    your work is open but not reproducible.
--   If you completely automate your analysis,
-    but your data is only available to people in your lab,
-    your work is reproducible but not open.
-
-The third key idea is [sustainability][sustainable-software].
-Software is sustainable if it's easier for people to maintain it and extend it than to replace it.
-However,
-sustainability isn't just a property of the software:
-it also depends on the skills and culture of its users.
-If a software package is being maintained by a couple of post-docs
-who are being paid a fraction of what they could earn in industry
-and have no realistic hope of promotion because their field doesn't value tool building,
-it doesn't matter whether the package is well tested and easy to install:
-sooner or later (probably sooner)
-it will become [abandonware][abandonware].
-
-### Why isn't all of this already normal?
-
-Nobody argues that research should be irreproducible or unsustainable,
-but "not against it" and actively supporting it are very different things.
-Academia doesn't yet know how to reward people for writing useful software,
-so while you may be thanked,
-the effort you put in may not translate into job security or decent pay.
-
-And some people still worry that if they make their data and code generally available,
-someone else will use it and publish a result they have come up with themselves.
-This is almost unheard of in practice,
-but that doesn't stop people using it as a scare tactic.
-Other people are afraid of looking foolish or incompetent by sharing code that might contain bugs.
-This isn't just [impostor syndrome][impostor-syndrome]:
-members of marginalized groups are frequently judged more harshly than others,
-so being wrong in public is much riskier for them.
+Please see Appendix \@ref(py-rse-install) for pointers on how to set all of this up.
```

Comment:
The details will be in the appendix, but do we need to allude to the fact that a bunch of data science libraries are needed (matplotlib, pandas, scipy, etc)? They all come with anaconda by default, but if you're not using anaconda you won't necessarily already have them.

Reply:
Modified - thanks.